src/modules/FileLocksmith/FileLocksmithUI/FileLocksmithXAML/Views/MainPage.xaml

```diff
@@ -195,7 +195,7 @@
         </Grid>
 
         <ContentDialog x:Name="SelectedFilesListDialog" x:Uid="SelectedFilesListDialog">
-            <ScrollViewer Padding="0,0,16,0">
+            <ScrollViewer Padding="0,0,16,0" HorizontalScrollBarVisibility="Auto">
                 <TextBlock IsTextSelectionEnabled="True" Text="{x:Bind ViewModel.PathsToString, Mode=OneWay}" />
```

Comment:
Should we add TextWrapping here as well so we can have multiple lines?

Reply:
@niels9001 That won't change anything though, because the scroll will make it scroll in a single line and not have multiple lines. Having both wrap and scroll won't fix anything here. Correct me if I am wrong.

Reply:
True, having both a scrollviewer + wrapping doesn't make sense. Either one or the other should work??

Reply:
I's say using TextWrapping is preferred then? Having to scroll to the end of the path is not great (and it hides the most important part of the path)? I think File Properties in explorer.exe does that as well?

Reply:
![image](https://github.com/microsoft/PowerToys/assets/92792319/f50fa186-e713-4eee-b658-cd52ece8d14d)
This is how that looks like.

Also, should we add a copy path and/or go-to path button?

Reply:
Oooh that looks good, I think this is more useful than having a scrollbar right? Agree on the copy/go-to path buttons? Maybe those could just be added to the dialog next to the close buttons? Or just next/under the path itself?

Reply:
Yeah, I guess this is better for the user for reasons you mentioned (personally prefer the scroll but that shouldn't matter).

I think, besides or below the path is more appropriate for the buttons than near the close button.

Reply:
@niels9001 working on that copy to clipboard button. Is creating the entire backbone for that including the helper functions the only way because that is a lot of code duplication. Or can I just reuse the functions from another module like the `ColorPickerUI`?